ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -3336,7 +3336,6 @@ mrm:
 	}
 }
 
-typedef ZEND_SET_ALIGNED(1, uint16_t unaligned_uint16_t);
```

Comment:
This line shouldn't have been removed? It breaks the build on Windows.

Reply:
I see now why you did this. https://github.com/php/php-src/commit/0e932f7ceaab503d136a524bf7f9cefb32be29fa should remove the need by no longer including zend_jit_gdb.c in the same file.